main.tex

```diff
@@ -594,15 +592,15 @@ \subsection{The special case of \(N=2\)}\label{sec:two_individuals}
 conclusions about zero determinant strategies however it is dependent on a
 specific structure of strategies. The analysis carried out here makes no
 assumptions about the structure of the strategies by using actual simulation
-interactions.
+interactions. \cite{Stewart...}
```

Comment:
Could you add this as a `% TODO` just to make sure we don't miss it, I guess you're wanting to add something here. The key is `Stewart2012`.